src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/convolution_backprop_data.cpp

```diff
@@ -23,8 +23,8 @@ const std::vector<std::vector<ptrdiff_t>> emptyOutputPadding = {{}};
 const std::vector<std::vector<ov::Shape>> inputShapes2D_static = {{{1, 3, 30, 30}},
                                                                   {{1, 16, 10, 10}},
                                                                   {{1, 32, 10, 10}}};
-const std::vector<std::vector<size_t>> kernels2D = {{1, 1}, {3, 3}, {3, 5}};
-const std::vector<std::vector<size_t>> strides2D = {{1, 1}, {1, 3}};
+const std::vector<std::vector<size_t>> kernels2D = {{3, 3}, {2, 2}};
```

Comment:
There would be some accuracy issue in the original case: 
 So removing kernel (1,1) . change kernel {3,5} to kernel {2,2} because most of 2d kernel should have same value on  space dimension. Also change stride for same reason. I don't know whether these tests parameters target some known special case or just some random number. Let us try restoring as it was anyway.

I did some analysis today after restoring.  but the case with *K(1.1)_*_PB(0.0)_PE(1.1)* would fail on accuracy check, which can't be reproduced in onednn. I would remove the PB(0.0)_PE(1.1). only keep  PB(0.0)_PE(0.0).  Usually padding should be same for the same axis...

Reply:
It indicates, that we have an accuracy issue for certain operation params, which may be an issue on the user side. Do we know the root cause of the issue? Do we know the limits of the issue (for which range of kernel sizes, padding, strides) we have accuracy issues? This is important to make a final decision on how exactly we handle this issue.